File: 0927/my_map.py
my_map = { "book": 10, "milk": 5 }
print(my_map["book"]) 
print(my_map["milk"])
my_map["hello"] = lambda: print("Hello from my_map!")
my_map["hello"]()
my_new_map = dict()
my_new_map["book"] = 10
my_new_map["milk"] = 5  
print(my_new_map["book"])
print(my_new_map["milk"])
my_new_map["hello"] = lambda: print("Hello from my_new_map!")
my_new_map["hello"]()

File: 0927/mymap.js
myMap = { book: 10, milk: 5 };
console.log(myMap.book); 
console.log(myMap.milk);
console.log(myMap["book"]);
console.log(myMap["milk"]);

myMap.hello = function() {
    console.log("Hello from myMap!");
};
myMap.hello();
myNewMap = new Map();
myNewMap.set("book", 10);
myNewMap.set("milk", 5);
console.log(myNewMap.get("book"));
console.log(myNewMap.get("milk"));

myNewMap.hello = function() {
    console.log("Hello from myNewMap!");
};
myNewMap.hello();